interfaces/HO-Portal/src/app/components/update-fsp/update-fsp.component.html

```diff
@@ -0,0 +1,74 @@
+<form
+  method="POST"
+  #updatePropertyForm="ngForm"
+  (ngSubmit)="doUpdate()"
+>
+  <ion-item>
+    <ion-label position="stacked">
+      {{ label }}
+    </ion-label>
+    <ng-container>
+      <ion-item class="ion-margin-top ion-align-self-stretch">
+        <ion-select
+          style="max-width: 100%;"
+          ngDefaultControl
+          name="propertyInput"
+          [(ngModel)]="propertyModel"
+          [value]="value"
+          [placeholder]="placeholder"
+          interface="popover"
+          (ionChange)="onFspChange($event)"
+        >
+          <ion-select-option
+            *ngFor="let fspItem of fspList"
+            value="{{ fspItem.fsp }}">
+              {{ fspItem.fsp }}
+          </ion-select-option>
+        </ion-select>
+      </ion-item>
+      <ng-container *ngIf="propertyModel != startingFspName">
+        <ion-text class="ion-padding-vertical" *ngIf="attributeDifference.length > 0">
+          {{ 'page.program.program-people-affected.edit-person-affected-popup.properties.fspChangeWarning' | translate }}:<br>
+          <div>
+            <span *ngFor="let attr of attributeDifference">{{ attr.name }} <span *ngIf="attr < attributeDifference.length -1">, </span></span>
+          </div>
+        </ion-text>
+        <ion-item *ngFor="let attribute of selectedFspAttributes" position="stacked">
+          <ion-label position="stacked">
+            {{ attribute.name }}
+          </ion-label>
+          <ion-input
+          ngDefaultControl
+          [name]="attribute.name"
+          [value]="attribute.value"
+          [placeholder]="attribute.placeholder?.en"
+          (ionChange)="onAttributeChange($event)"
+          ></ion-input>
+        </ion-item>
```

Comment:
Wouldn't it be possible to use the 'generic' "`<app-update-property-item>`"-component here?

That way all the same rules apply to all the fields...